src/store/authorization/actions.ts

```diff
@@ -35,20 +39,45 @@ export const actions: ActionTree<AuthorizationStateInterface, RootStateInterface
         return payload.errorCallback(r);
       });
   },
-  [AuthActionEnum.GET_DATA]: ({ commit }, payload) => {
-    HTTP.get(`/users/${payload}`).then((r: AxiosResponse<UserInterface>) => {
+  [AuthActionEnum.GET_DATA]: async ({ commit }, payload) => {
+    await HTTP.get(`/users/${payload}`).then((r: AxiosResponse<UserInterface>) => {
       commit(AuthMutationEnum.SET_USER, r.data);
     });
   },
-  [AuthActionEnum.UPDATE_USER]: ({ commit }, payload) => {
-    HTTP.get(`/users/${payload}`).then((r: AxiosResponse<UserInterface>) => {
-      commit(AuthMutationEnum.SET_USER, r.data);
-    });
 
-    commit(AuthMutationEnum.SET_USER, null);
-  },
   [AuthActionEnum.SIGN_OUT]: ({ commit, dispatch }, payload) => {
     dispatch('localStorageStore/REMOVE', 'user', { root: true });
     commit(AuthMutationEnum.SET_USER, payload);
   },
+
+  [AuthActionEnum.UPDATE_USER]: async ({ state, commit }, payload: UpdateUserInterface) => {
+    const payloadData = new UpdateUserModel(payload);
+    const userId = state.user!.id;
+    await HTTP.put(`/users/${userId}`, payloadData).then((r: AxiosResponse<UserInterface>) => {
+      commit(AuthMutationEnum.SET_USER, r.data);
+    });
+  },
+
+  [AuthActionEnum.DELETE_CONTACT]: async ({ state, commit, dispatch }, payload) => {
+    try {
+      const userId = state.user!.id;
+      await HTTP.delete(`/users/${userId}/contacts/${payload}`).then(() => {
+        commit(AuthMutationEnum.UPDATE_CONTACT, payload);
+      });
+    } catch (e) {
+      console.log(e);
+    }
+  },
+
+  [AuthActionEnum.ADD_CONTACT]: async ({ state, commit, dispatch }, payload) => {
+    const userId = state.user!.id;
+    const payloadData = new PostContactModel(payload[0]);
+    await HTTP.post(`/users/${userId}/contacts`, payloadData)
+      .then((r) => {
+        commit(AuthMutationEnum.ADD_CONTACT, r.data);
+      })
+      .catch((e) => {
+        console.log(e.response.data.error_message);
+      });
+  },
```

Comment:
Please, use one style of error handling in all actions

Reply:
ok, done!